plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -1222,6 +1234,11 @@ public AcidTransaction beginUpdate(ConnectorSession session, Table table, HiveUp
         return beginOperation(session, table, "update", AcidOperation.UPDATE, DataOperationType.UPDATE, Optional.of(updateProcessor));
     }
 
+    public AcidTransaction beginStatisticsCollection(ConnectorSession session, Table table)
+    {
+        return beginOperation(session, table, "analyze", AcidOperation.ANALYZE, DataOperationType.UPDATE, Optional.empty());
+    }
+
```

Comment:
why, below, we use `description + " in " + table.getSchemaTableName()` instead of `session.getQueryId()` for the `queryId` parameter in the `acquireTableWriteLock` call?

Reply:
This dates back to my first hive PR, Oct 15th, 2020, supporting `INSERT` and `DELETE` for transactional tables, Oct 15th, 2020 😞 

I changed `beginOperation` to pass the queryId instead of the funny string, and removed `description` from the parameters of helper methods `beginOperation` and `makeTransaction`.